docs/source/index.rst

```diff
@@ -3,31 +3,62 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
-Welcome to MetaSynth's documentation!
-=====================================
 
-MetaSynth is a package for generating statistical metadata from polars (or pandas) dataframes and creating synthetic data from this metadata.
 
-It can be used when private datasets cannot be shared directly. By creating a synthetic dataset,
-the data is safeguarded against the risk of disclosure. On the scale of disclosure/utilitly as defined
-by the `Office for National Statistics <https://www.ons.gov.uk/methodology/methodologicalpublications/generalmethodology/onsworkingpaperseries/onsmethodologyworkingpaperseriesnumber16syntheticdatapilot>`_,
-MetaSynth resides in the *Synthetic-augmented plausible* category. This means that individual columns
-should bear statistical/univariate resemblence to the original dataset, but the relationships between
-the columns in the dataset is lost.
+.. image:: /images/logos/blue.svg
+   :alt: MetaSynth_Logo
 
-To create a metadata file for your dataset, you need to create a polars dataframe. For a simple example
-see the :doc:`quick_start`.
+MetaSynth Documentation
+=======================
+Welcome to the MetaSynth documentation. MetaSynth is a Python package that facilitates the generation of synthetic data and metadata, enabling users to create surrogate datasets that mimic the structure and characteristics of their original data while preserving privacy.
 
+.. warning:: 
+   MetaSynth, and this documentation, are under active development. As a result, this documentation is still heavily work in progress. If you see any errors, missing content or have a suggestion for new content, feel free to :doc:`let us know </about/contact>`! 
+
+Documentation Outline
+---------------------
+
+This documentation is designed to help you easily navigate and find the information you need. It is organized into the following four sections:
+
+About
+^^^^^
+This section provides a brief overview of MetaSynth's purpose and functionality, contact information, frequently asked questions (FAQs), and licensing details.
+
+.. toctree::
+   :maxdepth: 2
+
+   about/about
+
+
+User's Guide
```

Comment:
It would be nice if these were clickable.

Reply:
Agreed!